### PR TITLE
remove cache-compiled-release command

### PR DESCRIPTION
**Pull request by crhntr**
The TAS tile has not used this command for a long time.

Also there is a better way to compile BOSH Releases now given https://github.com/cloudfoundry/bosh-agent/pull/315 has been merged. TAS Slingshots is working on workflows to do compilation for all tiles and those workflows will not use this command.

This command adds significant complexity to acceptance tests.

Removing it will simplify other work and will make kiln developers more likely to run acceptance tests more often and earlier.

### CODE REVIEW

**Comment by crhntr** [comment]
We have created an issue in Pivotal Tracker to manage this. Unfortunately, the Pivotal Tracker project is private so you may be unable to view the contents of the [story](https://www.pivotaltracker.com/story/show/186830626). 

The labels on this github issue will be updated when the story is started.

File: internal/acceptance/workflows/hello-tile
Comment (pvaramballypivot) [inline internal/acceptance/workflows/hello-tile:1]: @crhntr I have been avoiding pushing updates to this submodule as it gets picked up automatically. Could this commit cause issues?

File: internal/test/integration_test.go
Comment (pvaramballypivot) [inline internal/test/integration_test.go:1]: @crhntr and I discussed to remove these changes. Unrelated to this PR.

**Review by pvaramballypivot — APPROVED** [review]

File: internal/test/integration_test.go
Comment (pvaramballypivot) [reply internal/test/integration_test.go:1]: Done

File: internal/acceptance/workflows/hello-tile
Comment (pvaramballypivot) [reply internal/acceptance/workflows/hello-tile:1]: Done

**Comment by kvmw** [comment]
@crhntr @pvaramballypivot We used to run `cache-compiled-release` in our tile CI to cache the compiled releases (dependencies) in a s3 bucket, in order to speed up the next builds.  

Since this command has been removed, whats the alternative to cache the compiled dependencies, in order to avoid recompiling them for each CI execution ?

**Pull request merged**